openshift/templates/jenkins-persistent.json

```diff
@@ -316,6 +320,12 @@
       "displayName": "Jenkins ImageStreamTag",
       "description": "Name of the ImageStreamTag to be used for the Jenkins image.",
       "value": "jenkins:2"
+    },
+    {
+      "name": "FATAL_ERROR_LOG",
+      "displayName": "Fatal Error Log",
+      "description": "When a fatal error occurs, an error log is created with information and the state obtained at the time of the fatal error.",
```

Comment:
need to make it clearer that the valid values for this are true+false (pending my comment above about whether the path should be configurable)

Reply:
Will do.